staging/src/kubevirt.io/api/instancetype/v1beta1/types.go

```diff
@@ -330,12 +330,48 @@ type CPUPreferences struct {
 	//+optional
 	PreferredCPUTopology *PreferredCPUTopology `json:"preferredCPUTopology,omitempty"`
 
+	//
+	//+optional
+	SpreadOptions *SpreadOptions `json:"spreadOptions,omitempty"`
```

Comment:
I think Fabian has a point. All other fields also have `CPU` in their name. Do you think `CPUSpreadOptions` might make sense?

Reply:
I don't like `spec.cpu.cpuSpreadOptions`, maybe `spec.cpu.preferSpreadOptions`? 

IMHO we shouldn't duplicate `cpu` in the field names given the parent name, existing examples of this were a mistake.

Reply:
What about `preferredSpreadOptions`? Also, can we override spread options in an instancetype?

Reply:
> What about preferredSpreadOptions?

I suggested `preferSpreadOptions` as the associated `preferredCPUTopology` value for this is `preferSpread`.

I'm also fine with sticking with `spreadOptions` FWIW.

>  Also, can we override spread options in an instancetype?

No, the instance type is just providing an amount of resource in this context.

Reply:
Okay, let's keep it.